stan/math/prim/mat/fun/scale_matrix_exp_multiply.hpp

```diff
@@ -20,10 +20,11 @@ template <int Cb>
 inline Eigen::Matrix<double, -1, Cb> scale_matrix_exp_multiply(
     const double& t, const Eigen::MatrixXd& A,
     const Eigen::Matrix<double, -1, Cb>& B) {
+  if (A.size() == 0 && B.size() == 0)
```

Comment:
Same question here.

Reply:
This is the same as above, as the scaling by `t` is ininfluential.